taraxa/state/contracts/tests/dpos/dpos_test.go

```diff
@@ -129,6 +129,8 @@ var (
 			CornusHf: chain_config.CornusHfConfig{
 				BlockNum:                1000,
 				DelegationLockingPeriod: 4,
+				DagGasLimit:             100000,
+				PbftGasLimit:            1000000,
```

Comment:
There is the same default cfg in slashing_test.go. Should be added there same way as here